openmetadata-ui/src/main/resources/ui/src/components/AppRouter/AuthenticatedAppRouter.tsx

```diff
@@ -1346,6 +1346,9 @@ const AuthenticatedAppRouter: FunctionComponent = () => {
         )}
       />
       {routeElements}
+      <Route exact path={ROUTES.SIGNIN}>
+        <Redirect to={ROUTES.MY_DATA} />
+      </Route>
```

Comment:
@ShaileshParmar11 Can we cover other login-related routes like: `/register` `/signup`
If user is logged in we should redirect these to HOME only

Reply:
changes push